WTWSMS/common/cultures/00_cultures.txt

```diff
@@ -4644,7 +4645,13 @@ byzantine = {
 		
 		founder_named_dynasties = yes
 
-		modifier = default_culture_modifier
+		allow_looting = yes
+
+		character_modifier = {
+			martial = 4
```

Comment:
No but seriously now. First of all, never use Notepad for code editing. Secondly I will repeat my prior post:

> Why would it apply to other cultures too? 

As I said, and I will rephrase to make it very clear, the question is not about how you get the modifier or who gets it (only Spartans should as I said, and if you agree on that there is no contention), it is concerning the fact that the +4 martial is a too significant modifier for what it is meant to represent.